docs/linux/reporting_kernel_bugs.md

```diff
@@ -7,7 +7,12 @@ To find out the list of maintainers responsible for a particular kernel subsyste
 Make sure to mention the exact kernel branch and revision where the bug occurred.
 Many kernel mailing lists reject HTML formatted messages, so use the plain text mode when sending the report.
 
-Bugs without reproducers are way less likely to be triaged and fixed.
+Think of what you report. Today, Linux maintainers are overwhelmed with bug reports, so increasing the incoming flow won't help to fix all the bugs.
+The more actionable your report is, the higher the chance that it will be addressed.
+Note that people are more likely to care about kernel crashes (e.g. use-after-frees or panics) than of INFO: messages and such, unless it is clearly visible from the report what exactly is wrong.
+If there are stalls or hangs, only report them if they are frequent enough or have a reliable reproducer.
+
+Overall, bugs without reproducers are way less likely to be triaged and fixed.
```

Comment:
I would not phrase this this strongly. Fixing rate for bugs w/o reproducers is actually quite high. It really depends on the bug type and a number of other factors. I would at least remove "way".

Reply:
Note that this line was here before, I just changed "Bugs" to "Overall, bugs" to prettify the transition from the previous passage.

Reply:
ah, right